grpc/handlers.go

```diff
@@ -38,7 +40,7 @@ func (srv *grpcServer) SendTextileEvent(event events.TextileEvent) {
 }
 
 func (srv *grpcServer) ListDirectories(ctx context.Context, request *pb.ListDirectoriesRequest) (*pb.ListDirectoriesResponse, error) {
-	entries, err := srv.sv.ListDir(ctx)
+	entries, err := srv.sv.ListDir(ctx, "/")
```

Comment:
@dmerrill6 is this the default path ^^^?

Reply:
Actually, the implementation for `ListDir()` in space Service does not work with that path. I was meaning to ask you how the frontend traverses the directories, if we are not passing a path to the `ListDirectories` grpc call? @dmolina79

Reply:
ah ok then this is fine, i guess it could be empty `""` to mean the root of your space folder path, @dmerrill6 can confirm which one is better

Reply:
This is a bit confusing, not sure why you are modifying the `ListDir` signature if path not going to be used anyways. Currently the FE lists the whole directory structure (that's what `ListDir` is returning). There's also the `listDirAtPath` method that you might want to make public if you need to list a specific path. But yea default value to list root and all subdirs should be `""`.

Reply:
@dmerrill6 I just realized the actual behaviour of that function after discussing with Doug. I might take your advice and expose the `ListDirAtPath` as public, because it is currently wasteful for the native FileSystem to list all the nested directory tree on a single call. After taking a closer look at the implementation, I'll decide which one is more efficient and use it.